source/people-ops/onboarding.html.md

```diff
@@ -6,49 +6,12 @@ Congratulations on joining Nebulab! We're glad to have you aboard, and we can't
-faces to memorize, a lot of projects to explore and a lot of policies to learn. This is why we have
-developed an onboarding process that will help you get comfortable with how we do things around
-here.
+faces to memorize, a lot of projects to explore and a lot of policies to learn. Don't worry!
+Our onboarding process is fully automated and spans over a few weeks.
 
-## Choose your gear
+During these first weeks [Donut](https://www.donut.com), our friendly robot helper, will guide you
+through the intricacies of Nebulab and show you everything that you need to learn about both this
+[Playbook][playbook] as well as our [Internal Playbook][internal-playbook].
 
-The first thing you want to do is choose your hardware. We not only provide our employees with a
-MacBook Pro, but also a wide range of accessories.
-
-Head over to the [Benefits](/people-ops/benefits) page and make your choices!
-
-## Set up your tools
-
-Next up, you should set up your company accounts and install all the tools we use:
-
-- Make sure you can sign into your work Google account, and that everything's in order both in
-  Gmail and Google Calendar. You'll be using these extensively!
-- Use your Google account to sign into [Slack](https://nebulab.slack.com).
-- Install [Harvest](https://www.getharvest.com), sign in, and make sure you can see all
-  company-level projects and any client projects you've been assigned to.
-- Sign into [Factorial](https://factorialhr.com/) and maybe download their mobile
-  app too, so that you can easily submit expense reports when you're abroad.
-- Check that you're part of the [Nebulab GitHub organization](https://github.com/nebulab) and
-  update your company in your GitHub profile!
-
-If you need any help, don't be afraid to reach out to the person who handled your onboarding!
-
-## Meet your manager
-
-Your manager will coordinate your work and they will be responsible for your personal and
-professional growth. Our coaching practices are explained more thoroughly in
-[Coaching and 1:1s](/personal-growth/coaching-and-one-on-ones).
-
-When you join the company, your manager will reach out to schedule your first 1:1, where you'll get
-to know each other and structure your relationship together. They will also set up a call with the
-rest of your team, so that you can get to know the project and the people you'll be working with.
-
-## Read this playbook
-
-Finally, once you've completed all of the above, we suggest reading the rest of this playbook!
-
-You don't have to do it all in one day, or even in one week: simply take the time you need to go
-through the content and fully understand it.
-
-Take notes on any the aspects that are not clear and ask your manager to clarify them or, better 
-yet, [open a PR](/internal-projects/playbook)!
+[playbook]: https://playbook.nebulab.com
+[internal-playbook]: https://www.notion.so/nebulab/Internal-Playbook-d04a5791deee422ab4b0853119085bcb
```

Comment:
Not blocking, but maybe we want to link to [this paragraph](https://playbook.nebulab.com/internal-projects/playbook/#external-and-internal-playbook) instead? I see that outsiders tend to get confused when we link to "private" resources because they expect them to be accessible.

Reply:
💯 updating this asap